astro/cli/astro/cmd/flags_test.go

```diff
@@ -87,6 +87,12 @@ func TestPlanAllowedValues(t *testing.T) {
 		"staging",
 		"prod",
 	}
+	expectedModules := map[string][]string{
```

Comment:
Please merge input and output into table driven tests style, for example,
```go
testcases := []struct {
		msg      string
		input    string
		expected []string
	}{
		{
			msg:      "environment is dev",
			input:    "dev",
			expected: []string{"misc-dev", "test_env-dev"},
		},
		{
			msg:      "environment is mgmt",
			input:    "mgmt",
			expected: []string{"foo_mgmt-mgmt"},
		},
                // add additional test cases here
	}

	for _, tt := range testcases {
		t.Run(tt.msg, func(t *testing.T) {
			result := tests.RunTest(t, []string{
				"--config=merge_values.yaml",
				"plan",
				"--environment",
				tt.input,
			}, "fixtures/flags", tests.VERSION_LATEST)
			// Check that all expected modules were planned for environment env
			for _, module := range tt.expected {
				assert.Contains(t, result.Stdout.String(), module)
			}

			assert.Equal(t, 0, result.ExitCode)
		})
	}
```

Reply:
Done, please take another look.